### PR TITLE
Add new starter kit

### PR DESCRIPTION
**Pull request by amandeepmittal**
Add Gatsby + Bulma CSS starter kit.

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1,3 +1,19 @@
+- url: https://tender-raman-99e09b.netlify.com/
+  repo: https://github.com/amandeepmittal/gatsby-bulma-quickstart
+  description: A Bulma CSS + GatsbyJS Starter Kit
+  tags:
+    - Styling:Bulma
+    - Design
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
```

File: docs/starters.yml [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+    - Hackathon
+    - Landing Page
+    - Open Source
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
```

File: docs/starters.yml [hunk 3]
```diff
@@ -1,3 +1,19 @@
+- url: https://tender-raman-99e09b.netlify.com/
+  repo: https://github.com/amandeepmittal/gatsby-bulma-quickstart
+  description: A Bulma CSS + GatsbyJS Starter Kit
+  tags:
+    - Styling:Bulma
+    - Design
+    - Hackathon
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
```

**Comment by amandeepmittal** [comment]
@LekoArts I have followed your suggestion and removed them. Thank you.

**Comment by amandeepmittal** [comment]
Thank you @LekoArts @pieh 👍

**Pull request merged**